packages/documentation/docs/controls/forms/_forms-validation_styleguide.md

```diff
@@ -0,0 +1,45 @@
+Form validation gives a user feedback on their input to ensure accurate, consistent data submitted. If requirements aren’t met or data is incorrect, it’s rejected.
+
+![Invalid state](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=2767-5955&t=IIgjTqoOEP524yAH-4)
+
+Key aspects:
+
+- Data Accuracy: Collect precise information for informed decisions.
+- Security: Prevent malicious submissions.
```

Comment:
Unfortunately input validation cannot prevent malicious submissions because:

- The validation JavaScript code is running on the client and can be accessed or modified.
- The validation can be bypassed by sending data directly to the API.
- ...